app/controllers/pg_hero/home_controller.rb

```diff
@@ -23,7 +23,8 @@ class HomeController < ActionController::Base
     def index
       @title = "Overview"
       @extended = params[:extended]
-
+      @citus_enabled = @database.citus_enabled?
```

Comment:
I think we should add those lines with the PRs of each page not with this PR.

Reply:
Ok.